public/js/app.js

```diff
@@ -110,26 +121,68 @@ $(function(window, Observable){
   var selectedOptions = new Observable({
     region: 'London',
     burial: true,
+    cremation: false,
+    ordersheet: false,
+    notice: false,
+    flowers: false,
+    venue: false,
+    catering: false,
+    limousine: false
   });
 
   selectedOptions.onChange(calculateCost);
+  selectedOptions.onChange(addToSummaryList);
+
 
   function calculateCost() {
     var _this = this;
     var region = _this.region;
     var regionPrices = regionOptions[region];
     var cost = Object.keys(_this)
-      .filter(function(option){
-        return option !== 'region' && _this[option];
-      })
-      .reduce(function(price, option){
-        return price + regionPrices[option];
-      }, 0);
+    .filter(function(option){
+      return option !== 'region' && _this[option];
+    })
+    .reduce(function(price, option){
+      return price + regionPrices[option];
+    }, 0);
 
     $('#displayed-region').html(region);
     $('#cost').html('£' + cost);
+    $('.cost-breakdown').html('£' + cost);
   }
 
-  window.selectedOptions = selectedOptions;
+  function addToSummaryList() {
 
+    var list = $('#dynamic-list');
+    var listItems = [];
+    // TODO: Map keys to string name to avoid this horriffic if statement
+    for(var key in this){
+      if(this[key] === true && key === 'ordersheet'){
+        listItems.push('Order Sheets');
+      }
+      if(this[key] === true && key === 'notice') {
+        listItems.push('Funeral notice');
+      }
+      if(this[key] === true && key === 'flowers') {
+        listItems.push('Flowers');
+      }
+      if(this[key] === true && key === 'venue') {
+        listItems.push('Venue Hire');
+      }
+      if(this[key] === true && key === 'catering') {
+        listItems.push('Catering');
+      }
+      if(this[key] === true && key === 'limousine') {
+        listItems.push('Limousine');
+      }
+    }
+    list.html('');
+    listItems.map(function(el) {
+      list.append('<li>' + el + '</li>');
+    });
+  }
+
+  selectedOptions._fireHandlers();
```

Comment:
this line should be out. That method is going to be called automatically when any property of selectedOptions change.